.github/workflows/node.yaml

```diff
@@ -0,0 +1,38 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04, macos-latest, windows-latest]
+        node-version: [12.x, 13.x]
+        redis-version: [4, 5]
```

Comment:
I don't think we care about 4, but 6 was released yesterday, might be good to add that?